shift/rename_beesdoo.py

```diff
@@ -11,23 +11,138 @@
 
 _logger = logging.getLogger(__name__)
 
-MODELS_TO_RENAME = {
-    "beesddoo.shift.assign_super_coop": "shift.assign_super_coop",
-    "beesddoo.shift.generate_planning": "shift.generate_planning",
-    "beesddoo.shift.generate_shift_template.line": "shift.generate_shift_template.line",
-    "beesddoo.shift.generate_shift_template": "shift.generate_shift_template",
-    "beesdoo.shift.action_mixin": "shift.action_mixin",
-    "beesdoo.shift.daynumber": "shift.daynumber",
-    "beesdoo.shift.extension": "shift.extension",
-    "beesdoo.shift.holiday": "shift.holiday",
-    "beesdoo.shift.journal": "shift.journal",
-    "beesdoo.shift.planning": "shift.planning",
-    "beesdoo.shift.shift": "shift.shift",
-    "beesdoo.shift.subscribe": "shift.subscribe",
-    "beesdoo.shift.template": "shift.template",
-    "beesdoo.shift.temporary_exemption": "shift.temporary_exemption",
-    "beesdoo.shift.type": "shift.type",
+MODELS_TO_RENAME = [
+    ("beesddoo.shift.assign_super_coop", "shift.assign_super_coop"),
+    ("beesddoo.shift.generate_planning", "shift.generate_planning"),
+    (
+        "beesddoo.shift.generate_shift_template.line",
+        "shift.generate_shift_template.line",
+    ),
+    ("beesddoo.shift.generate_shift_template", "shift.generate_shift_template"),
+    ("beesdoo.shift.action_mixin", "shift.action_mixin"),
+    ("beesdoo.shift.daynumber", "shift.daynumber"),
+    ("beesdoo.shift.extension", "shift.extension"),
+    ("beesdoo.shift.holiday", "shift.holiday"),
+    ("beesdoo.shift.journal", "shift.journal"),
+    ("beesdoo.shift.planning", "shift.planning"),
+    ("beesdoo.shift.shift", "shift.shift"),
+    ("beesdoo.shift.subscribe", "shift.subscribe"),
+    ("beesdoo.shift.template", "shift.template"),
+    ("beesdoo.shift.temporary_exemption", "shift.temporary_exemption"),
+    ("beesdoo.shift.type", "shift.type"),
+]
+TABLES_TO_RENAME = [
+    ("beesddoo_shift_assign_super_coop", "shift_assign_super_coop"),
+    ("beesddoo_shift_generate_planning", "shift_generate_planning"),
+    (
+        "beesddoo_shift_generate_shift_template_line",
+        "shift_generate_shift_template_line",
+    ),
+    ("beesddoo_shift_generate_shift_template", "shift_generate_shift_template"),
+    ("beesdoo_shift_daynumber", "shift_daynumber"),
+    ("beesdoo_shift_extension", "shift_extension"),
+    ("beesdoo_shift_holiday", "shift_holiday"),
+    ("beesdoo_shift_journal", "shift_journal"),
+    ("beesdoo_shift_planning", "shift_planning"),
+    ("beesdoo_shift_shift", "shift_shift"),
+    ("beesdoo_shift_subscribe", "shift_subscribe"),
+    ("beesdoo_shift_template", "shift_template"),
+    ("beesdoo_shift_temporary_exemption", "shift_temporary_exemption"),
+    ("beesdoo_shift_type", "shift_type"),
```

Comment:
This violates DRY. I'd prefer to generate this half of the list from MODELS_TO_RENAME. Thoughts?